Cakefile

```diff
@@ -254,14 +252,40 @@ task 'doc:test', 'watch and continually rebuild the browser-based tests', ->
     fs.writeFileSync "#{outputFolder}/test.html", output
     log 'compiled', green, "#{testFile} → #{outputFolder}/test.html"
 
-  for target in [testFile, testsSourceFolder]
-    fs.watch target, interval: 200, renderTest
-  log 'watching...' , green
+  if watch
+    for target in [testFile, testsSourceFolder]
+      fs.watch target, interval: 200, renderTest
+    log 'watching...', green
+
+task 'doc:test', 'build the browser-based tests', ->
+  buildDocTests()
+
+task 'doc:test:watch', 'watch and continually rebuild the browser-based tests', ->
+  buildDocTests yes
+
+
+buildAnnotatedSource = (watch = no) ->
+  do generateAnnotatedSource = ->
+    exec "node_modules/docco/bin/docco src/*.*coffee --output docs/v#{majorVersion}/annotated-source", (err) -> throw err if err
```

Comment:
Most minor thing ever ... I know that you didn't change anything here, but I got curious. I always thought `./node_modules/.bin/whatever` was the way to go when invoking "binaries" installed locally with npm. I guess it doesn't matter. :)